slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -750,6 +786,12 @@ class SlideToActView @JvmOverloads constructor(
         }
     }
 
+    fun startAnimateTickIcon() {
```

Comment:
What's the reason behind this?

Reply:
The reason behind this is when i tried to put the the icon dynamically after completeAnimationEnd callback the new icon doesn't animate so i thought i could do this manually.

Reply:
I see. The problem is that this will expose an API publicly that we will have to maintain so I have to ask you to remove it.

Reply:
done